modules/gdscript/gdscript.h

```diff
@@ -83,6 +83,7 @@ class GDScript : public Script {
 	friend class GDScriptFunction;
 	friend class GDScriptAnalyzer;
 	friend class GDScriptCompiler;
+	friend class GDScriptDocGen;
```

Comment:
As mentioned above, if this needs to be a class for some scope access, then this is fine. But if it doesn't, then this is not needed either.

Reply:
See above :)